onboarding.md

```diff
@@ -108,8 +108,15 @@ The project has a venue for real-time discussion:
   * This is a repository to which all members of the `nodejs` GitHub
     organization (not just collaborators on Node.js core) have access. Its
     contents should not be shared externally.
+  * Node.js has a moderation team which you should contact when unsure
+    about taking action in the Node.js org.
+  * You can moderate non-collaborator posts yourself. When done please
+    report the moderation action taken in accordance to the moderation
+    policy.
```

Comment:
This is a good place to insert a link to the moderation policy. Additionally (and purely optionally), this is also a good place to mention that collaborators can volunteer for the moderation team and how to do that. (We may also want to try using language like "volunteer" rather than "self-nominate". I think people are more likely to volunteer for something than to nominate themselves for something. Plus _volunteer_ correctly implies that it's a service position whereas _nominate_ might suggest recognition, etc.)

Reply:
I think it might make sense to invert the order of things here maybe? Start with a title indicating this section is about moderation in general (rather than heated debates) and add things like volunteering under it here.